.github/workflows/ci-build.yaml

```diff
@@ -86,7 +86,19 @@ jobs:
             profile: minimal
           - test: test-python-sdk
             profile: minimal
+          - test: test-executor
+            install_k3s_version: v1.21.2+k3s1
```

Comment:
Hmm this is not right?

Reply:
This is the version were running

Reply:
We should be testing 1.24

Reply:
It defaults to latest (v1.25 ATM). These are for testing backwards compatibility.

Reply:
I see. Can we rename these tests to differentiate them from the existing ones? It might be better to explicitly set the version so that we know what change breaks the tests?

Reply:
Yes/no. I wast trying to avoid forcing all the required checks to be changed. That impacts all PRs, which would need to be synced with master.

Using latest reduces maintenance, we don’t need to update it.